src/core_read_parameter_file.c

```diff
@@ -62,8 +62,8 @@ int read_parameter_file(const int ThisTask, const char *fname, struct params *ru
     ParamAddr[NParam] = run_params->FileWithSnapList;
     ParamID[NParam++] = STRING;
 
-    strncpy(ParamTag[NParam], "LastSnapShotNr", MAXTAGLEN);
-    ParamAddr[NParam] = &(run_params->LastSnapShotNr);
+    strncpy(ParamTag[NParam], "LastSnapshotNr", MAXTAGLEN);
```

Comment:
Codacy found an issue: [Easily used incorrectly; doesn't always \0-terminate or check for invalid pointers [MS-banned] (CWE-120).  Risk is low because the source is a constant string.](https://app.codacy.com/manual/jacobseiler/sage-model/pullRequest?prid=6139772)